src/qol/increase-red-x-drops.s

```diff
@@ -0,0 +1,332 @@
+; Increases the probability of Red X drops to a minimum of 1/256. Any enemy that is already more likely to drop a Red X will not be affected.
+; The probability calculations are out of 1024 (0x0400). For X Drops this must add up precisely - so take from Yellow X chance 
+
+.org SpriteStats + Hornoad_Stats * SpriteStats_Size + SpriteStats_YellowXWeight
+    ; originally 337/C8/1
```

Comment:
Order isnt quite clear. I assume its yellow/green/red, right?
Would be nice to have that in the header comment somewhere